lang/chicken-scheme/Makefile

```diff
@@ -8,27 +7,38 @@
 
 include $(TOPDIR)/rules.mk
 
-PKG_NAME:=chicken-scheme-interpreter
-PKG_VERSION=5.1.0
+PKG_NAME:=chicken-scheme
+PKG_VERSION=5.2.0
 PKG_RELEASE:=1
 
-PKG_SOURCE:=chicken-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=https://code.call-cc.org/releases/$(PKG_VERSION)/
-PKG_HASH:=5c1101a8d8faabfd500ad69101e0c7c8bd826c68970f89c270640470e7b84b4b
 PKG_BUILD_DIR:=$(BUILD_DIR)/chicken-$(PKG_VERSION)
+PKG_SOURCE:=chicken-$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://code.call-cc.org/releases/5.2.0/
```

Comment:
Why?

Reply:
Sorry, should I use PKG_MAINTAINER? I thought that instead, I should use MAINTAINER insidethe package definition.